  lf: 0.12
  lr: 0.14
  m: 2.250
  Iz: 0.06
  Cf: 60.0
  Cr: 60.0
  mu: 0.1
  max_vel: 5.0
  min_vel: 0.1
